### DISCUSSION TITLE
installation issues

### DISCUSSION THREAD

**Discussion by jwierzch** [e-m-b-a/embark#223] [Q&A]
hey i see the installation bug was closed, but I am having issues and get either dev or default versions installed on ubuntu 22. Not sure whats going on, but seems to fail the emba installation, but if i cd into the emba directory and just run sudo install -d , emba installs fine. I am trying to install dev version to implement sso....

**Comment by jwierzch** [comment]
install logs
[install.txt](https://github.com/user-attachments/files/18027654/install.txt)


```ubuntu@embark-dev-1:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.5 LTS
Release:	22.04
Codename:	jammy

**Comment by BenediktMKuehne** [comment]
Is this relevant?
should we transfer this to a issue?
